widget/playground/src/helpers.ts

```diff
@@ -101,10 +101,12 @@ export function clearEmpties<T extends Record<string, any>>(obj: T): T {
 
 export function filterConfig(
   WidgetConfig: WidgetConfig,
-  initialConfig: WidgetConfig
+  initialConfig: WidgetConfig,
+  apiKey: string
```

Comment:
Because the API key provided by dApp should not use in playground because the default API key cors setting on server side doesn't allow it.
So it is only for the export modal input and exported code not for using in playground (api calls).

Reply:
Good point. How about handle it in `widget/playground/src/App.tsx`? 
We have something like this:
```
const config = useConfigStore.use.config();
<Route path="/*" element={<Widget config={config} />} />
```
we can change it to:
```
<Route path="/*" element={<Widget config={{...config, apiKey: OUR_PLAYGROUND_API_KEY}} />} />
```

In this way it's more explicit what we are doing here and it won't be missed in future refactors.